src/ui/components/Rating/index.js

```diff
@@ -68,9 +68,12 @@ export class RatingBase extends React.Component {
         id: `Rating-rating-${thisRating}`,
         key: `rating-${thisRating}`,
         ref: (ref) => { this.ratingElements[thisRating] = ref; },
+        title: rating ? i18n.sprintf(i18n.gettext(`Update your rating to %(thisRating)s out of 5.`), { thisRating }) :
```

Comment:
You aren’t checking for readOnly any,ore, so you’ll need to add that back. That’s why the tool tips are appearing in the small, read only versions.

Reply:
I moved out the check for `readOnly` to the if block that check for `readOnly` found [here](https://github.com/mozilla/addons-frontend/pull/4693/files#diff-88b641d04bd9c7d77b5475071f0870b6R76). The reason is that I could not nest ternary if statements and if I simply had `if (!readOnly && !rating) { ... }`, then tooltip would show `Update your rating to ... ` over the smaller read only stars.

This approach makes sure that the `readOnly` stars always have their `title` property removed and the clickable versions have a tooltip that dynamically reflects if the user is rating the add-on for the first or fifth time!

Reply:
For this check I'd set `rating && !readOnly`, because I don't think `readOnly` stars should each have the same duplicate title; if its `readOnly`, as mentioned below, we should just put a `title` attribute on the whole enclosing `<div>`.

Reply:
Thanks for the clarification - I actually removed the wrapping div as per your comment found [here](https://github.com/mozilla/addons-frontend/pull/4693#discussion_r178470860). Should we add it back and update it as necessary?

Reply:
Oh snap, I'm sorry. Yeah, I think it's worth having the wrapper `title` *only* when it's `readOnly`.